catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -260,4 +308,22 @@ private QueryRequest createQueryRequest(Filter filter) {
   public WorkspaceMetacardImpl getWorkspaceMetacard(String workspaceId) {
     return getWorkspaceMetacards(Collections.singleton(workspaceId)).get(0);
   }
+
+  @Override
+  public List<Metacard> getAssociationMetacards(String workspaceId) {
+    Query query = new QueryImpl(getAssociationFilter(workspaceId));
+    QueryRequest queryRequest = new QueryRequestImpl(query);
+
+    return ResultIterable.resultIterable(catalogFramework::query, queryRequest)
+        .stream()
+        .map(Result::getMetacard)
+        .filter(Objects::nonNull)
+        .collect(Collectors.toList());
+  }
+
+  private Filter getAssociationFilter(String workspaceId) {
+    return filterBuilder.allOf(
+        filterBuilder.attribute(Core.METACARD_TAGS).is().like().text(ASSOCIATION_TAG),
```

Comment:
:question: should we want `.like()` included in these? I think it would work fine without them

Reply:
To be honest I'm not sure if `like()` is needed however I've seen it used in other places